src/main/rx/rx.c

```diff
@@ -403,6 +404,15 @@ STATIC_UNIT_TESTED uint16_t updateLinkQualitySamples(uint16_t value)
     sampleIndex = (sampleIndex + 1) % LINK_QUALITY_SAMPLE_COUNT;
     return sum / LINK_QUALITY_SAMPLE_COUNT;
 }
+
+void setRFmodeDirect(uint8_t rf_mode_value)
```

Comment:
Probably better to bound the entire function with `#ifdef USE_RX_LINK_QUALITY_INFO` and also make sure the calls are properly excluded.

Reply:
I'm not sure what this means? add it to the unit tests?

Reply:
No, not unit tests. The entire function is already bounded by `#ifdef USE_RX_LINK_QUALITY_INFO` so just change to:
```
void setRFmodeDirect(uint8_t rf_mode_value)
{
    rf_mode = rf_mode_value;
}
```